code/game/objects/items/storage/storage.dm

```diff
@@ -37,6 +37,9 @@
 	var/use_sound = "rustle"	//sound played when used. null for no sound.
 	var/opened = 0 //Has it been opened before?
 	var/list/content_watchers = list() //list of mobs currently seeing the storage's contents
+	/// A list of items already in the storage at the creation of it
+	var/list/obj/item/starting_items
```

Comment:
I do not like this list as it should at best be anulled immediately after use since it will create large lists per object

Reply:
I can't annull it, because i need it in loadouts.